### DISCUSSION TITLE
Weird info message when not using @InputArgument

### DISCUSSION THREAD

**Discussion by elizeusdsantos** [Netflix/dgs-framework#837] [General]
I just created a Data fetcher without the annotation `@InputArgument` for the arguments. The parameters are being well assigned but I can see some info messages in my console which I suppose are not necessary.

The data fetcher query:

```
    @DgsQuery
    public Iterable<Location> findLocations(Integer count) {
        return myService.findLocations(count);
    }
```
and here the "info" message:

`"message":"Unknown argument 'count' on data fetcher my_datafetcherclass_here.my_method"`

```
The parameter is being well assigned because it has the same name that I have in my Query declaration:
    findLocations(
        count: Int)
    : [Location!]
```

This message should appear only when my parameter has a different name of my input name used in my query declaration. In this case the binding would be impossible unless using `@InputArgument(name = "the_name_here")`

**Comment by berngp** [comment]
You make an excellent point, thanks for reporting it.

**Comment by paulbakker** [comment]
The fact that it works without `@InputArgument` is unintended/accidental behavior. We obviously need to be clear about the expected behavior though, thanks for reporting!

@berngp We have to figure out if supporting arguments without `@InputArgument` makes sense, and what the limitations would be.

**Comment by berngp** [comment]
I am in favor of annotating the argument explicitly. I don't think we should officially support it and instead express a clear error if we are left with arguments that we, the framework, could not bind.
